frontend/src/constants/queryBuilderQueryNames.ts

```diff
@@ -6,6 +6,8 @@ type QueryParamNames =
 	| 'selectedFields'
 	| 'linesPerRow';
 
+export type QuerySearchParamNames = 'viewName' | 'viewKey';
```

Comment:
this is not needed in the global level as this is needed in the save view module

Reply:
Okay;.